nearcore/src/config.rs

```diff
@@ -862,25 +888,29 @@ pub fn init_configs(
 ) -> anyhow::Result<()> {
     fs::create_dir_all(dir).with_context(|| anyhow!("Failed to create directory {:?}", dir))?;
 
+    let chain_id = chain_id
+        .and_then(|c| if c.is_empty() { None } else { Some(c.to_string()) })
```

Comment:
quite confusing..

Do we need to support this case (Some("")) ? - consider adding an onboarding/good-first-issue to clean it up.